### ISSUE TITLE
Missing models/ source code required for reproducing training losses

### ISSUE THREAD

**Issue by QuHuaimin** [Daniil-Selikhanovych/RSD#5]
Hi, thank you very much for releasing the RSD code and pretrained model.

I would like to kindly ask about the missing models/ Python source directory.

As also discussed in #4, the current repository does not contain the models/ package, while configs/RSD.yaml directly depends on it, for example:

model:
  target: models.unet.UNetModelSwin
  target_gan: models.unet.UNetModelSwinGanWrapper

diffusion:
  target: models.script_util.create_gaussian_diffusion

In #4, the initial reply seems to have interpreted models/ as the directory for ResShift checkpoint weights. However, the issue reporter later clarified that the missing part is the Python source code itself (models.script_util, models.unet, etc.).

This also affects training reproduction rather than only inference. In particular, without the original diffusion/model implementation, it is difficult to directly reuse and faithfully reproduce the official training_student_losses() logic and its associated loss computation. For example, when reproducing or modifying the LPIPS loss, we currently have to implement a workaround instead of calling the original training loss implementation.

Would it be possible to release the missing models/ source directory, or alternatively point us to the exact upstream repository/commit from which these files should be taken?

This would be very helpful for faithfully reproducing both the training procedure and the reported loss settings.

Thank you again for your work and for your time!

**Comment by Daniil-Selikhanovych** [comment]
Hello, @QuHuaimin 
Thanks for your notice. I apologize for my misclick while pushing the code when I forgot the folder "models". I pushed the code for this folder now. Let me know if the problem disappears.

**Comment by QuHuaimin** [comment]
Thank you very much for the quick update! This is very helpful to me.
